src/lib/licenses.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Licenses recognized by the OSI (Open Source Initiative).
+ * This is a list of their SPDX license identifiers.
```

Comment:
their website is kind of bad.... the list had a bunch of duplicates and I found it to be missing some of the common ones on Github which I tried to add